docs/README.md

```diff
@@ -1253,13 +1253,9 @@ export class HandleAvailability {
 }
 ```
 
-#### Events ordering
-
-<!-- TODO: several people have asked about how Booster ensures event ordering. I think it makes sense to explain that here  -->
-
 ### 3. Event handlers
 
-In event-driven architectures we have different parts of our application that react to events. In the case of Booster, we have the entities (in charge of reducing the events), and the _event handlers_. These are classes decorated with the `@EventHandler` decorator whose goal is to execute business logic or trigger other events when a specific event occurs.
+In event-driven architectures we have different parts of our application that react to events. In the case of Booster, we have the entities (in charge of reducing the events), and the _event handlers_. These are classes decorated with the `@EventHandler` decorator which can be defined as reactive side effect of the registration of a certain event, that can be used to execute some business logic or register other events related to the handler.
```

Comment:
As expected with event-driven architectures, multiple parts of our application react to events. In the case of Booster, we have entities (in charge of reducing the events) and _event handlers_. These last ones are classes decorated with the `@EventHandler` decorator. Every time a new instance of a given event is registered, the `handle` method of this class is triggered. This method can contain any business logic defined by the user or it can also register new events.

I think this explains the concept better, what do you think?